features/manifest.yml.dist

```diff
@@ -0,0 +1,110 @@
+# Generated from: manifest.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: false
+  support:
+    chrome: "96"
+    chrome_android: "96"
+    edge: "96"
+compat_features:
+  # baseline: false
+  # support:
+  #   chrome: "39"
+  #   chrome_android: "39"
+  #   edge: "79"
+  #   firefox_android: "79"
+  #   safari: "17"
+  #   safari_ios: "11.3"
+  - html.manifest.name
+  - html.manifest.short_name
+  - html.manifest.start_url
```

Comment:
I'd prefer to split it in two instead. One general manifest feature that captures only the more widely available things, knowing that it doesn't unlock all that much dev scenarios, and knowing that it won't be baseline unless Firefox desktop decides to care about PWAs again.
And, another feature that captures the chrome/edge-only things. I'll push a commit to do this.

Reply:
Splitting it up makes sense. Re-request a review when it's done?

Reply:
Doing more research on this. Here's my analysis so far:

* The browsers we care about here are Chrome, Chrome for Android, Edge, Firefox for Android, Safari, and Safari for iOS.

  * Firefox on desktop is not in this list because it doesn't support web app manifests at all as far as I know. This is expected.

* These keys are supported in all the above browsers, so should be kept in the main manifest feature:

  * `html.manifest.name`
  * `html.manifest.short_name`
  * `html.manifest.start_url`
  * `html.manifest.icons`
  * `html.manifest.theme_color`
  * `html.manifest.scope`

* `html.elements.link.rel.manifest` should absolutely be in the main manifest feature, and supported in the above browsers. It's missing in Safari according to BCD.

  * I think is a mistake, which I'm fixing in https://github.com/mdn/browser-compat-data/pull/24971

* `html.manifest.display` should also be supported in the same browsers, but because a couple of display types are missing in Safari, the key is marked as partial in BCD, which is a problem for us.

  * Trying to fix it in https://github.com/mdn/browser-compat-data/pull/24970

* `html.manifest.id` is missing on Firefox Android, and my vague recollections of past `id`-related discussions tell me that `id` isn't yet a scenario that developers care about.

  * We should probably ignore this one for now.

* `html.manifest.orientation` and `html.manifest.background_color` are not in Safari or Safari iOS.

  * I also tend to think we should ignore these. `background_color` is a feature that's used to give a PWA window some background color before the stylesheet has loaded, therefore avoiding flashes of white while an app is loading. `orientation` lets you define the default orientation of your app when it launches. While nice features, we could separate them later if needed.

* `html.manifest.description` (and `html.manifest.screenshots`, but this one is not in BCD) are supplemental members that are not made for user agents necessarily, but rather for stores that want to list PWAs. The only thing that uses them right now is the "rich install dialog" in Chrome.

  * I think we should ignore for now.

* `html.manifest.serviceworker.*` is, apparently for web payment.

  * This belongs in a separate feature, and I don't have enough subject matter expertise to work on it, so I'll ignore it for now.